omod/src/main/webapp/pages/registerPatient.gsp

```diff
@@ -305,6 +305,9 @@ ${ ui.includeFragment("uicommons", "validationMessages")}
                                 configOptions.formFieldName = field.formFieldName
                                 configOptions.left = true
                                 configOptions.classes = field.cssClasses
+                                if (field.type == 'personAttribute') {
+                                    configOptions.uuid = field.uuid
```

Comment:
I'd think that you should be including the personAttribute fragment if type is personAttribute

Reply:
it is included anyway, this gsp page generates fieldsets for fragments from the config file, where we define that there will be personAttribute fragment